admin-dev/themes/new-theme/js/pages/module/controller.js

```diff
@@ -1124,6 +1134,44 @@ class AdminModuleController {
       self.updateModuleVisibility();
     });
   }
+
+  updateTotalResults() {
+    // If there are some shortlist: each shortlist count the modules on the next container.
+    const $shortLists = $('.module-short-list');
+    if ($shortLists.length > 0) {
+      $shortLists.each(function shortLists() {
+        const $this = $(this);
+        updateText(
+          $this.find('.module-search-result-wording'),
+          $this.next('.modules-list').find('.module-item').length
+        );
+      });
+
+      // If there is no shortlist: the wording directly update from the only module container.
+    } else {
+      const modulesCount = $('.modules-list').find('.module-item').length;
+      updateText($('.module-search-result-wording'), modulesCount);
+
+      if (self.currentDisplay === self.DISPLAY_LIST) {
+        $(this.addonItemListSelector).toggle(modulesCount !== (this.modulesList.length / 2));
+      } else {
+        $(this.addonItemGridSelector).toggle(modulesCount !== (this.modulesList.length / 2));
+      }
```

Comment:
I think this could be simplified like this:

```js
const selectorToToggle = (self.currentDisplay === self.DISPLAY_LIST)? this.addonItemListSelector : this.addonItemGridSelector;
$(selectorToToggle).toggle(modulesCount !== (this.modulesList.length / 2));
```

Reply:
Done. Thanks!